build.xml

```diff
@@ -72,7 +72,7 @@
   <property name="jni.revision" value="1"/>
   <property name="jni.build" value="0"/> <!--${build.number}-->
   <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
-  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
+  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
```

Comment:
I’ll deal with the native rebuilds.  I have VM setups for a number of them and access to some compile farms for the others.  Build and test on whichever you have available; I’ll build/test the rest after merge.

> On Feb 5, 2016, at 11:19 AM, marco2357 <notifications@github.com> wrote:
> 
> In build.xml:
> 
> > @@ -72,7 +72,7 @@
> >    <property name="jni.revision" value="1"/>
> >    <property name="jni.build" value="0"/> <!--${build.number}-->
> >    <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
> > -  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
> > +  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
> 
> Updated the PR (bumped major version from 4 to 5).
> I don't think I can rebuild the natives myself unless there's some fancy cross compilation option in JNA I don't know about. I only have Linux x64. How can I do this?
> 
> —
> Reply to this email directly or view it on GitHub.
>